src/dmd/target.d

```diff
@@ -65,13 +65,13 @@ extern (C++) struct Target
     uint classinfosize;       /// size of `ClassInfo`
     ulong maxStaticDataSize;  /// maximum size of static data
 
-    // C ABI
+    /// C ABI
     TargetC c;
 
-    // C++ ABI
+    /// C++ ABI
     TargetCPP cpp;
 
-    // Objective-C ABI
+    /// Objective-C ABI
```

Comment:
Is this a leftover?

Reply:
Kinda. But it is a separate, trivial commit.
I can make it its own PR if you want, just didn't think it was worth the CI run.